### ISSUE TITLE
arXiv-2013/03-Studio Ousia's Quiz Bowl Question Answering System

### ISSUE THREAD

**Issue by BrambleXu** [BrambleXu/knowledge-graph-learning#143]
**一句话总结：**

**资源：**

- [pdf](https://arxiv.org/abs/1803.08652)
- [wikipedia2vec](https://github.com/wikipedia2vec/wikipedia2vec)
- [paper-with-code](

**关键字：**

- dataset: 
- 

**笔记：**

题目的例子：

![image](https://user-images.githubusercontent.com/10768193/57002993-60b74700-6bfe-11e9-9f57-416eb434a999.png)

# 2 Proposed system

## 2.1 Data

使用了多个数据来源。
- used the question-answer pairs available at the Protobowl website，其中包含了10万的问题和答案组。
-  public QA dataset provided by Iyyer et al. 包含2万个问题答案
- 并且使用了wiki和Freebase作为外部的数据来源

### 2.2.1 Model

整体架构：
![image](https://user-images.githubusercontent.com/10768193/57003018-9d833e00-6bfe-11e9-99ab-17c3c67ed3f1.png)

Architecture of Neural Quiz Solver
![image](https://user-images.githubusercontent.com/10768193/57003030-b68bef00-6bfe-11e9-8ba4-b3f54d2441fa.png)

### 2.2.2 Entity detection

使用entity linking自动标注entity names. 方法基于keyphraseness [6], which is the probability that an entity name is used as an anchor in Wikipedia.

### 2.2.3 Pretrained representations

To initialize the vector representations of words (pw), entities (qe), and answers (ae),
we use Wikipedia2Vec4[7]. 

### 2.2.4 Other details

The minibatch size is fixed as 32, the learning rate is automatically controlled by Adam [8]
> [8] Diederik Kingma and Jimmy Ba. Adam: A Method for Stochastic Optimization. arXiv preprint arXiv:1412.6980v9, 2014.
> 这篇论文很久了，我找了下关于[权值衰减的文章](https://zhuanlan.zhihu.com/p/52084949)

## 2.3 Neural Type Predictor

We use the FIGER entity type set [10], which consists of 112 fine-grained entity types, as the target
entity types. 

We use two separate models with the following different target entity types: all fine-grained entity types and only eight coarse-grained entity type. 当做是多分类人物。

## 2.4 Information retrieval models

As others have in past studies [2, 13, 14], we use conventional IR models to enhance the performance of our QA system. In particular, we compute multiple relevance scores against the documents associated with the target answer using the words in a question as a query.

## 2.5 Answer Scorer

Given a question as an input, the Answer Scorer assigns a relevance score to each answer based on the outputs of the neural network models and IR models described above. 

然后，使用GBRT来把问题变成一个二分类问题，判断哪个答案最相关。这里使用了微软的lightGBM

**模型图：**

**结果**：

**接下来要看的论文**：
